### DISCUSSION TITLE
PSI is a rogue service provider

### DISCUSSION THREAD

**Discussion by aamirparkar** [community/community#50652] [GitHub Education]
Is there a way Github could use some reliable service provider for certification exams?
Their call-centre is a dead end as no-one wants to resolve your issues. Feels like you are paying them for nothing and then charging the certification amounts from candidates.

**Comment by LuizCampedelli** [comment]
I totally agree, i am also faccing issues with PSI, besides you have to call them, in 2024, you cant do it online, by their own website, nobody answer or fix anything in their call-centre, they push you to GH support, i had an international call of 17 minutes to resolve nothing at all.

Dont ever asked for accommodations in the Psi exam form, if you do it, your account is blocked!!!!
